tensorflow_datasets/text/indic_corpus/indic_corpus.py

```diff
@@ -0,0 +1,101 @@
+"""indic_corpus dataset."""
+import tensorflow_datasets as tfds
+
+_DESCRIPTION = """
+Description is **formatted** as markdown.
+
+It should also contain any processing which has been applied (if any),
+(e.g. corrupted example skipped, images cropped,...):
+"""
+
+_CITATION = """
+@inproceedings{kakwani2020indicnlpsuite,
+    title={{IndicNLPSuite: Monolingual Corpora, Evaluation Benchmarks and Pre-trained Multilingual Language Models for Indian Languages}},
+    author={Divyanshu Kakwani and Anoop Kunchukuttan and Satish Golla and Gokul N.C. and Avik Bhattacharyya and Mitesh M. Khapra and Pratyush Kumar},
+    year={2020},
+    booktitle={Findings of EMNLP},
+}
+"""
+
+VERSION = tfds.core.Version("1.0.0")
+
+RELEASE_NOTES = {
+    "1.0.0": "Initial release.",
+}
+
+INDIC_LANGS = ["as", "bn", "en", "gu", "hi", "kn", "ml", "mr", "or", "pa", "ta", "te"]
+
+
+class IndicCorpusConfig(tfds.core.BuilderConfig):
+    """BuilderConfig for IndicCorpus"""
+
+    def __init__(self, name, languages, **kwargs):
+        super().__init__(name=name, version=VERSION, **kwargs)
+
+        self.languages = languages
+
+
+class IndicCorpus(tfds.core.GeneratorBasedBuilder):
+    """DatasetBuilder for indic_corpus dataset."""
+
+    BUILDER_CONFIGS = [
+        IndicCorpusConfig(
+            "indic_corpus",
+            languages=INDIC_LANGS,
+            description="Indic Corp has 12 languages and is generated from web sources",
+        )
+    ]
+
+    def _info(self) -> tfds.core.DatasetInfo:
+        """Returns the dataset metadata."""
+        features = {"text": tfds.features.Text()}
+        return tfds.core.DatasetInfo(
+            builder=self,
+            description=_DESCRIPTION,
+            features=tfds.features.FeaturesDict(features),
+            homepage="https://indicnlp.ai4bharat.org/home/",
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: tfds.download.DownloadManager):
```

Comment:
@sumanthd17 Using the `tfds.download` here gives an Attribute error on windows:
`AttributeError: partially initialized module 'tensorflow_datasets' has no attribute 'download' (most likely due to a circular import)`
This can simply be written as `dl_manager` without specifying its type explicitly.